util/find-up.test.ts

```diff
@@ -51,5 +51,5 @@ Deno.test('findUp', async (t) => {
     assertEquals(ret, null);
   });
 
-  Deno.remove(root, { recursive: true });
+  await Deno.remove(root, { recursive: true });
```

Comment:
@shawn-render added this to fix the flaky test runs because otherwise the tests would fail in many cases with ```

findUp => ./util/find-up.test.ts:10:6
error: Leaking async ops:
  - 1 async operation to op_fs_remove_async was started in this test, but never completed. The operation was started here:
    at handleOpCallTracing (ext:core/01_core.js:610:42)
    at Object.opAsync (ext:core/01_core.js:603:15)
    at Object.remove (ext:deno_fs/30_fs.js:199:14)
    at file:///Users/tzvifriedman/dev/render-cli/util/find-up.test.ts:54:8
    at eventLoopTick (ext:core/01_core.js:183:11)
    at async innerWrapped (ext:cli/40_testing.js:544:5)
    at async asyncOpSanitizer (ext:cli/40_testing.js:193:27)
    at async resourceSanitizer (ext:cli/40_testing.js:426:25)
    at async exitSanitizer (ext:cli/40_testing.js:475:27)
    at async outerWrapped (ext:cli/40_testing.js:489:14)
```

Reply:
Thank you! I was just looking into that too but you beat me to it 😄